api.yaml

```diff
@@ -322,6 +322,76 @@ paths:
               #! end !#
               "bodyPassthrough": true
             }
+
+  /ops/pgsql/v1/schema/version/up:
+    get:
+      summary: "Migrate database schema one version up"
```

Comment:
awesome that we are exposing these (up/down) 
however, I did notice in the documentation for migrate that the Up() will migrate "applying all up migrations." If we make sure not to merge any migration scripts into the code base that we aren't ready to run then I think this is fine. But, similarly, the Down() will migrate "applying all down migrations" and I'm not sure we will want to go back that far if we find one of the "up" migrations unsuccessful. There are other methods that we could expose (in addition to these or instead of) to run the migrations (Steps() to indicate relative up/down migrations or Migrate() if we want to feed it a specific version) and still maintain the de-coupling from app deployment

Reply:
Fixing, thank you!